aws/PAS-AIO-network-environment-template.json

```diff
@@ -1,841 +0,0 @@
-{
```

Comment:
Why do we remove AIO network?

Reply:
That was a decision made in v10.3 not release this network, it is not secured according to CyberArk standard.